### PR TITLE
added thanks to creatorlist

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

File: src/dashboard/components/CreatorList.vue [hunk 1]
```diff
@@ -78,6 +81,7 @@ export default {
   created() {
     this.$store.dispatch('db/ensureCreators').then(() => {
       this.loading = false;
+      console.log(this.creators);
```
Comment (dwilid) [inline src/dashboard/components/CreatorList.vue]: remove?

Comment (ErikBjare) [reply src/dashboard/components/CreatorList.vue]: Ah, yes

**Comment by ErikBjare** [comment]
@patrik-laurell The behavior is the same for time spent. I'm fine with that being intended behavior since otherwise, it would require doing database queries at some fixed interval (which would consume resources even if the tab is in the background) or create some fancy listener which is too much work tbh.

**Comment by patrik-laurell** [comment]
A db reload wherever you go to the dashboard could be ok?

**Comment by ErikBjare** [comment]
I don't think it is given how minor the 'bug' is.

**Pull request merged**